### PR TITLE
Update restore-deleted-group.md

### PR DESCRIPTION
**Pull request by msbemba**
Corrected Admin to administrator

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1542

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/create-groups/restore-deleted-group.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ If the deleted group doesn't appear here, contact an administrator.
 
 ## Restore a group in the Microsoft 365 admin center
 
-If you are a global administrator or a groups admin, you can restore a deleted group in the Microsoft 365 admin center:
+If you are a global administrator or a groups administrator, you can restore a deleted group in the Microsoft 365 admin center:
 
 1. Go to the admin center at [https://admin.microsoft.com](Go to the admin center at https://admin.microsoft.com).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/create-groups/restore-deleted-group.md]:
```suggestion
1. Go to the [admin center](https://admin.microsoft.com).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@MikePlumleyMSFT - PR has been copy edited and is ready for final review. This is based on issue #1542. 

Thank you.

cc: @kenwith

**Pull request merged**